docs/source/markdown/podman-image-scp.1.md

```diff
@@ -0,0 +1,69 @@
+% podman-image-scp(1)
+
+## NAME
+podman-image-scp - Securely copy an image from one host to another
+
+## SYNOPSIS
+**podman image scp** [*options*] *name*[:*tag*]
+
+## DESCRIPTION
+**podman image scp** copies container images between hosts on a network. You can load to the remote host or from the remote host as well as in between two remote hosts.
+Note: `::` is used to specify the output file on the host or the image name depending on if you are saving or loading.
+
+**podman image scp [GLOBAL OPTIONS]**
+
+**podman image** *scp [OPTIONS] NAME[:TAG] HOSTNAME[::FILELOCATION]*
+
+**podman image** *scp [OPTIONS] HOSTNAME[::IMAGENAME]*
+
+**podman image** *scp [OPTIONS] HOSTNAME[::IMAGENAME] HOSTNAME[::]*
+
+## OPTIONS
+
+#### **--quiet**, **-q**
+
+Suppress the output
+
+#### **--help**, **-h**
+
+Print usage statement
+
+## EXAMPLES
+
+
+```
+$ podman image image scp alpine
+Loaded image(s): docker.io/library/alpine:latest
+```
+
+```
+$ podman image scp alpine Fedora::/home/charliedoern/Documents/alpine
+Getting image source signatures
+Copying blob 72e830a4dff5 done
+Copying config 85f9dc67c7 done
+Writing manifest to image destination
+Storing signatures
+Loaded image(s): docker.io/library/alpine:latest
+```
+
+```
+$ podman image scp Fedora::alpine Fedora::
```

Comment:
This also seems not very useful — show this with two different connection names?